Makefile

```diff
@@ -2,13 +2,14 @@ build: export LUA_LIB_NAME=lua
 build: export LUA_LIB=$(CURDIR)/lua/lua5.1/
 build: export LUA_INC=$(CURDIR)/lua/lua5.1/include
 build:
-	cargo build
+	cargo build --features hot-reload
+	powershell copy target/debug/dcs_grpc_server.dll target/debug/dcs_grpc_server_hot_reload.dll
```

Comment:
`dcs_grpc_server.dll` and `target/debug/dcs_grpc_server_hot_reload.dll` will be the same with the difference that DCS loads and locks `dcs_grpc_server_hot_reload.dll`, while `dcs_grpc_server.dll` in return loads `dcs_grpc_server_hot_reload.dll`. If a mission is stopped, `dcs_grpc_server.dll` release its lock on `dcs_grpc_server_hot_reload.dll`, which in the end allows to re-compile the code between mission restarts instead of having to restart DCS>

Reply:
I do my testing in a local single-player game of DCS and as long as the mission is not running I don't actually have this issue as far as I remember.

However having the extra ability is a great idea.

Reply:
> I do my testing in a local single-player game of DCS and as long as the mission is not running I don't actually have this issue as far as I remember.

Me too. Previously, unloading the mission also unlocked the DLL. But once the hook env loads the DLL, it locks it until DCS is closed. So we didn't have the issue previously, but the hook addition would have introduced it if it weren't for the hot reloading.

Reply:
Aaaah right, gotcha.